notification/jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  preset: 'ts-jest',
-  testEnvironment: 'node',
-  roots: ['./src']
-}
+    preset: 'ts-jest',
+    testEnvironment: 'node',
+    roots: ['./tests']
```

Comment:
unit tests in js typically are in src directory and follow the pattern `{fileName}.test.js`... so here `./src` would be the right value